src/scripts.js

```diff
@@ -97,6 +115,145 @@ function getUserById(id, listRepo) {
   return listRepo.getDataFromID(id);
 }
 
+function windowOnClick(event) {
+  if (event.target.classList.contains('new-hydration-button')) {
+    submitNewHydration(userNowId);
+  }
+  if (event.target.classList.contains('new-activity-button')) {
+    submitNewActivity(userNowId);
+  }
+  if (event.target.classList.contains('new-sleep-button')) {
+    submitNewSleep(userNowId);
+  }
+}
+
+function getTodaysDate() {
+  let today = new Date();
+  let dd = String(today.getDate()).padStart(2, '0');
+  let mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  let yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+function submitNewHydration(userNowId) {
+  // let todaysDate = getTodaysDate();
+  let todaysDate = '2020/9/31';
+  /*
```

Comment:
I am about to submit a PR that has all these comments cleaned up for all the functions in script that have to do with users being able to submit data (POSTing).

Reply:
Sounds perfect, I may have to have you explain this.